### ISSUE TITLE
ER_PARSE_ERROR: You have an error in your SQL syntax

### ISSUE THREAD

**Issue by mrikirill** [strapi/strapi#280]
Hi!

Have created connection with local mysql 5.7.19 and created 1 content type with one field title via admin panel. But the new table did't appear in database and when I tried to add data, the console shown an error: 

```
ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1
    Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1
```

**Node.js version**: v8.6.0
**npm version**: 5.4.2
**Strapi version**: 3.0.0-alpha.6.3
**Operating system**: macOS 10.13

**Comment by Aurelsicoko** [comment]
@mrikirill  Can you give me a filename or more details about the error? Moreover, please give me the steps to reproduce the error by myself. Thanks!

**Comment by mrikirill** [comment]
Sure)

1.  Install Strapi (3.0.0-alpha.6.3) npm i strapi@lates -g and start new project
2. Go to admin panel and create connection with local MySQL database (5.7.19)
3. Receive error in console that I need to start command: npm i mysql. Install it and start strapi again
4. Go to admin panel re-create connection with MySQL (it connected ok)
5. Create content Account with filed name (string) save it
6. There is no table in database
7. Go to admin panel and try to add data in Account. Admin panel return error: 500 internal error
8. Console shows: 
ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1
    Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1

**Comment by mrikirill** [comment]
Repeated all steps today, the same error.
Add screenshots of errors:

Step 3: https://screencast.com/t/a37vMmy9e
Step 7: https://screencast.com/t/MWlgZm4kvRzM
Step 8: https://screencast.com/t/VMckQ3KrvvS

I guess it will be helpful

**Comment by Aurelsicoko** [comment]
@mrikirill Thanks for these details! Did you create the table in your database? What happens if you create one?

**Comment by hadihallak** [comment]
can confirm the same error on windows 10

**Comment by mrikirill** [comment]
@Aurelsicoko the same error after I've created the table with the same fields like I did it via the admin panel

**Comment by Aurelsicoko** [comment]
Thanks for your feedback! I will take a closer look tomorrow.

**Comment by bbjansen** [comment]
Same error here with 3.0.0-alpha.6.3 and v8.7.0. When I add a MySQL DB and click the content type link I get:

`DEBUG (8773 on ThinkBob): GET /content-manager/explorer/articles?skip=0&limit=10&sort=id (4 ms)
ERROR (8773 on ThinkBob): ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1
    Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SCHEMA 'public'' at line 1
        at Query.Sequence._packetToError (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/sequences/Sequence.js:52:14)
        at Query.ErrorPacket (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
        at Protocol._parsePacket (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/Protocol.js:279:23)
        at Parser.write (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/Parser.js:76:12)
        at Protocol.write (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/Protocol.js:39:16)
        at Socket.<anonymous> (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/Connection.js:103:28)
        at emitOne (events.js:115:13)
        at Socket.emit (events.js:210:7)
        at addChunk (_stream_readable.js:264:12)
        at readableAddChunk (_stream_readable.js:251:11)
        at Socket.Readable.push (_stream_readable.js:209:10)
        at TCP.onread (net.js:587:20)
        at Protocol._enqueue (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/protocol/Protocol.js:145:48)
        at Connection.query (/home/bob/projects/cryptoanalysis/dashboard/node_modules/mysql/lib/Connection.js:208:25)
        at afterCreate (/home/bob/projects/cryptoanalysis/dashboard/node_modules/strapi-knex/lib/index.js:109:20)
    From previous event:
        at /home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/client.js:242:72
        at runCallback (timers.js:785:20)
        at tryOnImmediate (timers.js:747:5)
        at processImmediate [as _immediateCallback] (timers.js:718:5)
    From previous event:
        at Object.create (/home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/client.js:239:39)
        at Pool._createResource (/home/bob/projects/cryptoanalysis/dashboard/node_modules/generic-pool/lib/generic-pool.js:354:17)
        at Pool.dispense [as _dispense] (/home/bob/projects/cryptoanalysis/dashboard/node_modules/generic-pool/lib/generic-pool.js:314:10)
        at Pool.acquire (/home/bob/projects/cryptoanalysis/dashboard/node_modules/generic-pool/lib/generic-pool.js:436:8)
        at /home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/client.js:289:19
    From previous event:
        at Client_MySQL.acquireConnection (/home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/client.js:280:12)
        at /home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/runner.js:208:30
    From previous event:
        at /home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/runner.js:206:35
    From previous event:
        at Runner.ensureConnection (/home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/runner.js:205:34)
        at Runner.run (/home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/runner.js:51:42)
        at Builder.Target.then (/home/bob/projects/cryptoanalysis/dashboard/node_modules/knex/lib/interface.js:35:43)
        at runCallback (timers.js:785:20)
        at tryOnImmediate (timers.js:747:5)
        at processImmediate [as _immediateCallback] (timers.js:718:5)
`

**Comment by Aurelsicoko** [comment]
@bbjansen It has been fixed, a patch will be released tomorrow.

**Comment by Aurelsicoko** [comment]
This issue has been fixed on [v3.0.0@alpha.6.4](https://github.com/strapi/strapi/releases/tag/v3.0.0-alpha.6.4).
